src/main/java/de/medizininformatikinitiative/torch/model/management/CachelessResourceBundle.java

```diff
@@ -19,7 +19,7 @@ public record ImmutableResourceBundle(
     /**
      * Creates an immutable version of a given ResourceBundle.
      */
-    public ImmutableResourceBundle(ResourceBundle bundle) {
+    public CachelessResourceBundle(ResourceBundle bundle) {
```

Comment:
you need to copy the maps in the compact constructor. otherwise it's still possible to create a CachelessResourceBundle with mutable maps. This constructor should just call the compact constructor.